api/adaptive_load/input_variable_setter.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package nighthawk.adaptive_load;
+
+import "google/protobuf/any.proto";
+
+// Selects and configures an InputVariableSetter plugin, which encapsulates
+// knowledge of how to apply a varying numeric input value to a
+// CommandLineOptions proto. For example, RequestsPerSecondInputVariableSetter
+// aka "rps" writes the value to the CommandLineOptions |requests_per_second|
+// field. Some InputVariableSetter plugins such as "rps" will not define a
+// config proto, in which case typed_config should be unset.
+message InputVariableSetterConfig {
+  // Name used to locate the plugin in the Envoy registry.
```

Comment:
Wherever we refer to a plugin name - it would be good to also include some indication where the user can find a list of available plugins of the kind.